### PR TITLE
feat: update ACA-Py to 0.8.2-rc0

### PR DESCRIPTION
**Pull request by anwalker293**
Tests are passing, but I wasn't able to manually test the tunnels tests. (Becoming unresponsive on my machine?) I'll ask someone on our team if they can run this, but opening PR now :)

### CODE REVIEW

**Comment by anwalker293** [comment]
Need to find viable image of ngrok version 3

**Comment by anwalker293** [comment]
@dbluhm

**Comment by anwalker293** [comment]
Working now! :tada:

File: docker-compose.tunnels.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   alice:
-    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-indy-1.16.0-0.8.1-rc2
+    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-indy-1.16.0-0.8.2-rc0
```
Comment (dbluhm) [inline docker-compose.tunnels.yml:4]: 0.8.2 is now officially released :slightly_smiling_face: 
```suggestion
    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-indy-1.16.0-0.8.2
```

**Review by dbluhm — CHANGES_REQUESTED** [review]
Let's target the official release

**Review by dbluhm — APPROVED** [review]

**Pull request merged**